src/crypting/Main.java

```diff
@@ -0,0 +1,42 @@
+package crypting;
+
+import crypting.strategy.UnicodeCipher;
+
+public class Main {
+
+    private static CliParser parser;
+    private static Cryptor cryptor;
+
+    public static void main(String[] args) {
+        parser = new CliParser(args);
+        cryptor = new Cryptor();
+
+        if ("unicode".equals(parser.optionOf("-alg")))
+            cryptor.setCipher(new UnicodeCipher());
+
+        String cryptedData = getCryptedOf(readData());
```

Comment:
Read data from what?

Reply:
Fixed

Reply:
What does "fixed" mean? I need to know that you understand why I asked it